core/src/main/java/com/rappytv/toolwarn/listener/ConfigMigrationListener.java

```diff
@@ -55,6 +56,7 @@ private void migrateFromOne(ConfigurationVersionUpdateEvent event) {
             pickaxe = config.get("pickaxePercentage").getAsInt();
             axe = config.get("axePercentage").getAsInt();
             shovel = config.get("shovelPercentage").getAsInt();
+            bow = config.get("bowPercentage").getAsInt();
```

Comment:
This needs to be removed as it would throw an exception resulting in every other value not being migrated
```suggestion
```